src/test/java/com/philips/research/spdxbuilder/core/domain/ConversionInteractorTest.java

```diff
@@ -124,37 +127,51 @@ void setsDocumentIdentification() {
     }
 
     @Nested
-    class spdxDocument{
+    class spdxDocument {
         ByteArrayOutputStream spdxOutputStream = new ByteArrayOutputStream();
         Stream<String> lines;
+        DateTimeFormatter isoFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'")
+                .withZone(ZoneId.of("UTC"));
+        String dateString = "2000-01-01T01:01:01.111Z";
+        DateTimeFormatter parser = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+        LocalDateTime localDateTime = LocalDateTime.parse(dateString, parser);
+
         @BeforeEach
         void setUp() {
+            bom.setCreatedAt(dateString);
             bom.addPackage(project).addPackage(pkg);
+            bom.setCreatedAt("2000-01-01T01:01:01.105Z");
```

Comment:
why setting twice?

This will also change if we update to have a typed param here.